kernel/packages/shared/social/hotScenes.ts

```diff
@@ -4,10 +4,17 @@ import { StoreContainer } from 'shared/store/rootTypes'
 import { fetchSceneIds } from 'decentraland-loader/lifecycle/utils/fetchSceneIds'
 import { fetchSceneJson } from 'decentraland-loader/lifecycle/utils/fetchSceneJson'
 import { SceneJsonData } from 'shared/types'
+import { reportScenesFromTiles } from 'shared/atlas/actions'
 import { getSceneNameFromAtlasState, postProcessSceneName } from 'shared/atlas/selectors'
 
 declare const globalThis: StoreContainer
 
+declare const window: {
+  unityInterface: {
+    UpdateHotScenesList: (info: HotSceneInfo[]) => void
+  }
+}
+
 type CrowdedSceneRealmInfo = {
```

Comment:
Why we have the concept of `CrowdedScenes` here? Maybe it would unify the naming better if we call this type `HotSceneRealmInfo`?